storage/registers.go

```diff
@@ -16,9 +16,14 @@ type RegisterIndex interface {
 	Get(ID flow.RegisterID, height uint64) (flow.RegisterValue, error)
 
 	// LatestHeight returns the latest indexed height.
-	LatestHeight() uint64
+	// Expected errors:
+	// - storage.ErrNotFound if the latest height wasn't set.
```

Comment:
This should never happen unless the register index is not bootstrapped, which we should return another sentinel error, such as `storage.ErrNotBootstrapped`